<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameLobby</class>
 <widget class="QDialog" name="GameLobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="User1">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>160</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="User2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>270</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="User3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>360</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="User4">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>450</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rankButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>ranking</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="level">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="waitButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>WaitingRoom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>430</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>User1</zorder>
  <zorder>User2</zorder>
  <zorder>User3</zorder>
  <zorder>User4</zorder>
  <zorder>rankButton</zorder>
  <zorder>name</zorder>
  <zorder>level</zorder>
  <zorder>waitButton</zorder>
  <zorder>updateButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
